libtextsecure/message_receiver.js

```diff
@@ -28,6 +29,7 @@ function MessageReceiver(username, password, signalingKey, options = {}) {
   this.signalingKey = signalingKey;
   this.username = username;
   this.password = password;
+  this.server = WebAPI.connect({ username, password });
```

Comment:
WebAPI? this is too generic to explain what it is...

Reply:
Yeah it's not the most clear name... It's a Signal file which we actually removed all the references too early on because it was all code for contacting Signal servers, I just left put this initialiser back in to leave the attachment code mostly untouched but it definitely might make more sense to just move the put/get attachment stuff to a loki specific file and let the web api wither away?